Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -17,24 +19,40 @@ void SFilledGraphWidget::Construct(const FArguments& InArgs)
 			.OnImGuiDrawDelegate(this, &SFilledGraphWidget::OnImGuiDraw)
 		];
 
-	HandleDataArray();
+	HandleDataArrays();
 }
 
-void SFilledGraphWidget::HandleDataArray()
+void SFilledGraphWidget::HandleDataArrays()
 {
-	Algo::Sort(InData2D, [](const FDataPoint& LeftDataPoint, const FDataPoint& RightDataPoint)
-		{
-			return LeftDataPoint.X < RightDataPoint.X;
-		});
+	XMaxValue = DataArrays[0][0].X;
+	YMaxValue = DataArrays[0][0].Y;
+	YMinValue = 0.0f;
 
-	for (FDataPoint Point : InData2D)
+	for (TArray<FDataPoint> CurrentDataArray : DataArrays)
 	{
-		XValues.Add(Point.X);
-		YValues.Add(Point.Y);
-	}
+		Algo::Sort(CurrentDataArray, [](const FDataPoint& LeftDataPoint, const FDataPoint& RightDataPoint)
```

Comment:
Nice use of the Algo library! I think the Sort.h header should be included at the top of the file though